src/components/pages/ServiceReleaseProcess/components/ServiceDetails.tsx

```diff
@@ -77,6 +81,31 @@ export default function ServiceDetails() {
     }
   }
 
+  const getImageUrl = () => {
+    if (fetchServiceStatus) {
+      return `${getAssetBase()}/api/administration/documents/${fetchServiceStatus.leadPictureId}`
+    }
+    return `${getAssetBase()}/images/content/ServiceMarketplace.png`
+  }
+
+  useEffect(() => {
+    async function fetchImage() {
+      try {
+        const actualImage = await fetchImageWithToken(getImageUrl())
+        const firstByte = Buff2Hex(actualImage.slice(0, 1))
+        const first3Bytes = Buff2Hex(actualImage.slice(0, 3))
+        const imageType =
+          IMAGE_TYPES[firstByte] ?? IMAGE_TYPES[first3Bytes] ?? 'image/*'
+        setActualImage(
+          URL.createObjectURL(new Blob([actualImage], { type: imageType }))
+        )
+      } catch (error) {
+        console.error('An error occurred while fetching the image:', error)
```

Comment:
for images I suggest not to output an error to the console but simply show a fallback image